README.md

```diff
@@ -74,6 +74,76 @@ perfplot log_Array1k_19-09-2019_12-13-49
 
 This will generate a file called `log_Array1k_19-09-2019_12-13-49.pdf`.
 
+# Save results to SQL database
```

Comment:
Put that into a separate file and link from here to not blow up this readme file too much.

Reply:
done